jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -875,3 +875,99 @@ def _fft_rng_factory(dtype):
                                np.array([0, 0xFFFFFFFF], dtype=np.uint32),
                                np.array([0xFFFFFFFF, 0xFFFFFFFF], dtype=np.uint32)])
 )
+
+lax_conv_general_dilated_general = tuple(
```

Comment:
How long do these tests take to run? Is there a way to reduce the number of tests by thinking which of the features are orthogonal?